remixer_core/src/test/java/com/google/android/libraries/remixer/InitializationHelper.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.libraries.remixer;
+
+import com.google.android.libraries.remixer.DataType;
+import com.google.android.libraries.remixer.Remixer;
+import java.util.HashSet;
+
+/**
+ * Helper class to properly initialize remixer instances.
+ */
+public class InitializationHelper {
+
+  public static void init() {
```

Comment:
This is a test helper, the real version is in RemixerInitialization and it does a few more things (initialize default UI mappings and so on).

You already filed a bug to catch uninitialized remixer instances early, I'm thinking of some way to do this, but will follow up on this on that bug.